msgcntr/messageforums-component-impl/src/java/org/sakaiproject/component/app/messageforums/entity/TopicEntityProviderImpl.java

```diff
@@ -375,7 +375,6 @@ public void deleteEntity(EntityReference ref, Map<String, Object> params) {
 				}
 
 				for (DiscussionForum forum : forums) {
-					if(forum.getDraft().equals(Boolean.FALSE)){
```

Comment:
Does this mean everyone who calls the entitybroker interface will see all the draft topics, even if they don't have access to them?